docs/contributor/questions.rst

```diff
@@ -0,0 +1,140 @@
+.. _contquestions:
+
+Contributing practice questions
+================================
+As a contributor, you are suggesting questions for :ref:`skills <skill>` learned by the student. These questions form part of the review material that learners will use to practice that specific skill. 
+
+Watch the video to learn how to contribute questions, or if you prefer reading the instructions instead, keep scrolling down.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/qUP8bc2izTg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+1. From the Community Dashboard, go to the **Submit Question** menu header. Here you will see a list of skills (and their Topics) to which you can start contributing questions.
```

Comment:
Since we have to update it later on anyway (when users get the option to select a specific topic), shall we leave it for now and update it when this changes?

Reply:
Sounds good!